karaf/apache-brooklyn/pom.xml

```diff
@@ -108,6 +108,31 @@
     </resources>
 
     <plugins>
+
+        <plugin>
+            <artifactId>maven-resources-plugin</artifactId>
+            <executions>
+                <execution>
+                    <id>copy-notice</id>
+                    <phase>package</phase>
+                    <goals>
+                        <goal>copy-resources</goal>
+                    </goals>
+                    <configuration>
+                        <outputDirectory>${basedir}/target/assembly</outputDirectory>
+                        <resources>
+                            <resource>
+                                <directory>${basedir}/target/classes/META-INF</directory>
+                                <includes>
+                                    <include>NOTICE</include>
+                                </includes>
```

Comment:
The `NOTICE` file created in `target/classes/META-INF` contains
```
Brooklyn Karaf Distribution
Copyright 2012-2016 The Apache Software Foundation

This product includes software developed at
The Apache Software Foundation (http://www.apache.org/).
```

while Brooklyn's files contain:

```
Apache Brooklyn
Copyright 2014-2016 The Apache Software Foundation

This product includes software developed at
The Apache Software Foundation (http://www.apache.org/).
```

I believe this will copy Karaf's `NOTICE` in Brooklyn's distribution.
You can copy the file along the existing `LICENSE` from elsewhere in the source tree. No need to maven foo.